distributed/comm/tests/test_ucx_config.py

```diff
@@ -20,32 +20,58 @@
 @pytest.mark.asyncio
 async def test_ucx_config(cleanup):
 
-    ucx = {"nvlink": True, "infiniband": True, "net-devices": ""}
+    ucx = {
+        "nvlink": True,
+        "infiniband": True,
+        "net-devices": "",
+        "tcp": True,
+        "cuda_copy": True,
+    }
 
     with dask.config.set(ucx=ucx):
         ucx_config = _scrub_ucx_config()
         assert ucx_config.get("TLS") == "rc,tcp,sockcm,cuda_copy,cuda_ipc"
         assert ucx_config.get("NET_DEVICES") is None
 
-    ucx = {"nvlink": False, "infiniband": True, "net-devices": "mlx5_0:1"}
+    ucx = {
+        "nvlink": False,
+        "infiniband": True,
+        "net-devices": "mlx5_0:1",
+        "tcp": True,
+        "cuda_copy": False,
+    }
 
     with dask.config.set(ucx=ucx):
         ucx_config = _scrub_ucx_config()
-        assert ucx_config.get("TLS") == "rc,tcp,sockcm,cuda_copy"
+        assert ucx_config.get("TLS") == "rc,tcp,sockcm"
         assert ucx_config.get("NET_DEVICES") == "mlx5_0:1"
 
     ucx = {
         "nvlink": False,
         "infiniband": True,
         "net-devices": "all",
         "MEMTYPE_CACHE": "y",
+        "tcp": True,
+        "cuda_copy": True,
     }
 
     with dask.config.set(ucx=ucx):
         ucx_config = _scrub_ucx_config()
         assert ucx_config.get("TLS") == "rc,tcp,sockcm,cuda_copy"
         assert ucx_config.get("MEMTYPE_CACHE") == "y"
 
+    ucx = {
+        "nvlink": False,
+        "infiniband": False,
+        "net-devices": "all",
+        "MEMTYPE_CACHE": "y",
+        "tcp": False,
+        "cuda_copy": True,
+    }
+    with dask.config.set(ucx=ucx):
+        with raises(ValueError):
```

Comment:
Did `raises` get imported earlier or should this be `pytest.raises`? Seeing [this linting error in `master`]( https://travis-ci.org/dask/distributed/jobs/657551806#L460 ).

```
distributed/comm/tests/test_ucx_config.py:72:14: F821 undefined name 'raises'
```

Reply:
Maybe this fixes it ( https://github.com/dask/distributed/pull/3541 )?

Reply:
Thanks for the fixes John.  I think something is not syncing correctly between my vscode sessions and the remote node. Again, thank you for fixing my mistake

Reply:
No worries 😉